app/build.gradle

```diff
@@ -36,7 +36,7 @@ apply plugin: "org.jlleitschuh.gradle.ktlint"
 
 android {
     compileSdkVersion 31
-    buildToolsVersion '32.0.0'
+    buildToolsVersion '33.0.0'
```

Comment:
This change seems unrelated to the PR

Reply:
Yes, which is why I put it in a separate commit, will remove it 👍